cluster-up/cluster/kind-k8s-sriov-1.17.0/manifests/sriovdp-config.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sriovdp-config
+  namespace: kube-system
+data:
+  config.json: |
+    {
+      "resourceList": [{
+        "resourceName": "sriov_net",
+        "selectors": {
+          "drivers":["vfio-pci"],
+          "pfNames": ["enp4s0f0","enp4s0f1"]
```

Comment:
this won't work on all machines, does it ?
locally i had to change it.

Reply:
It is patched by config_sriov.sh later on, I added those interfaces names so one could have a sense of how it should look like.
I picked those names from what I saw on CI nodes

Reply:
I suggest then to have a place holder instead them both, and envsubst it, wdyt?
it will be more self explanatory and maintainable

Reply:
I am ok with using `envsubst` for now